### DISCUSSION TITLE
Add Support for S3000-G2

### DISCUSSION THREAD

**Discussion by frankol** [nathanmarlor/foxess_modbus#470] [Ideas]
Hi, 
i got a protocol description of this inverter but cant get it running. i cant share the document in public, but could provide it to @nathanmarlor 
how can i send it to you?
i can test it instantly since the inverter isnt mounted yet :-)

**Comment by frankol** [comment]
the S-series is exactly like the t-series, so i guess its not the right place for me

**Reply by canton7** [reply]
I'm afraid this integration supports the H-series (and derivatives) only.

Someone started looking into the S-series, see #231, but I don't think they got very far. There was also some discussion on the Facebook group, which IIRC didn't make much progress either.
